docs/guide/slots/legend.md

```
@@ -0,0 +1,58 @@
+# `legend`  <Badge type="warning" text="deprecated" vertical="middle" />
+
+In this slot you can put an additional element that you want to display beside the circle legend.
+This is simply a legacy slot from the time when the [**`default`**](#default) slot did not exist yet.
+It can be helpful in some situations, but is much less powerful.
+
+### Usage 📜
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)